### PR TITLE
Create components

### PR DESCRIPTION
**Pull request by lqt93**

### CODE REVIEW

**Comment by huytd** [comment]
Sorry dude, look like there is a good amount of conflict here, since we just moved to moai.js :joy:

**Comment by lqt93** [comment]
Solved conflicts

File: components/Entry/index.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Link from 'next/link';
+import Skeleton from 'react-loading-skeleton';
+import styles from './Entry.module.css';
```
Comment (xuanvan229) [inline components/Entry/index.tsx:4]: Should we move it to the bottom?

File: components/Entry/index.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  return (
+    <div style={{ marginBottom: '32px', padding: '22px', background: '#FFFFFF', borderRadius: 'var(--radius-1)' }}>
+      <h1><Skeleton /></h1>
+      <Skeleton count={5} />
+    </div>
+  )
+}
+export const Entry = ({ doc }) => {
+  return (
+    <div style={{ marginBottom: '32px', padding: '22px', background: '#FFFFFF', borderRadius: 'var(--radius-1)' }}>
```
Comment (xuanvan229) [inline components/Entry/index.tsx:25]: Should we move style to CSS?

File: pages/index.tsx [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
 import Parser from 'rss-parser';
 import LazyLoad from 'react-lazyload';
-import Skeleton from 'react-loading-skeleton';
-import { Button, DivPx, Tag } from '@moai/core';
-import { HiOutlineExternalLink as externalLink } from 'react-icons/hi';
+import Layout from "../components/Layout";
+import { Entry, LoadingEntry } from '../components/Entry';
+import { Button } from '@moai/core';
 
 const FEEDS = [
```
Comment (xuanvan229) [inline pages/index.tsx:10]: unuse variable

Comment (lqt93) [reply pages/index.tsx:10]: It is used here:

https://github.com/webuild-community/federated-blog/pull/1/files#diff-aa98fd0757d0e1741503c50cfafb7726939d19819638bbe8e030a27adfec34a3R26

Comment (xuanvan229) [reply pages/index.tsx:10]: ah my bad

File: components/Entry/index.tsx (hunk 1 above)
Comment (lqt93) [reply components/Entry/index.tsx:4]: Actually, we dont have any code convention at the moment, you should propose it in issue first:
https://github.com/webuild-community/federated-blog/issues/2

File: components/Entry/index.tsx (hunk 2 above)
Comment (lqt93) [reply components/Entry/index.tsx:25]: yeah, thanks for reminding, I will move it to module.css

Comment (huytd) [reply components/Entry/index.tsx:25]: Apparently the right way to do is to use Moai's `Pane` component, but that pane don't support customizing the border and border radus, so I use this instead.

The right way to do in my opinion would be either customize Moai's Pane and use it here, or create a new Pane component and also use it here.

File: components/Entry/Entry.module.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  margin: 10px;
+  padding: 10px;
+}
+
+.readMoreArea {
+  display: flex;
+  gap: 10px;
+}
+
+.primaryButton {
```
Comment (huytd) [inline components/Entry/Entry.module.css:19]: These button classes are not used anymore, we should remove it

File: components/Entry/Entry.module.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
+.tag {
```
Comment (huytd) [inline components/Entry/Entry.module.css:1]: This one is not used anymore too

File: components/Entry/index.tsx [hunk 6]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Link from 'next/link';
+import Skeleton from 'react-loading-skeleton';
+import styles from './Entry.module.css';
+import { Button, Tag } from '@moai/core';
+import { HiOutlineExternalLink as externalLink } from 'react-icons/hi';
+
+const linkSymbol = <svg width="14px" height="14px" viewBox="0 0 24 24" className="css-1ctnorc"><g id="external_link" className="icon_svg-stroke" stroke="#666" strokeWidth="1.5" fill="none" fillRule="evenodd" strokeLinecap="round" strokeLinejoin="round"><polyline points="17 13.5 17 19.5 5 19.5 5 7.5 11 7.5"></polyline><path d="M14,4.5 L20,4.5 L20,10.5 M20,4.5 L11,13.5"></path></g></svg>;
```
Comment (huytd) [inline components/Entry/index.tsx:8]: Hmm, I removed this icon already

**Comment by huytd** [comment]
@lqt93 right now

**Comment by lqt93** [comment]
@huytd solved all conflicts, please merge it before you merge any of your new features

**Comment by huytd** [comment]
Sorry dude, I see more conflict :joy:

**Comment by lqt93** [comment]
@huytd  Please take a look at this PR firstly T_T

**Review by huytd — APPROVED** [review]

**Pull request merged**